ch5-catdir/controller.cpp

```diff
@@ -55,6 +74,10 @@ void Controller::process(const std::string_view& path) try
     auto dir = Directory(path);
     while (auto ent = dir.next())
     {
+        if (!isArgument)
+        {
+            m_out << '\t';
+        }
```

Comment:
IIUC, you add a single tab before the info, only for subdirectories not given as an argument in the command-line.
What about subdir of a subdir? Do you want them to be nested further (with 2 tabs) or just 1 tab for all of them?